#!/bin/sh -e
## 20_missing_newlines.dpatch by Andreas Metzler <ametzler@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add missing newline to help-messages of locate and xargs

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
diff -urNad /home/ametzler/TIN/FINDUTILS/findutils-4.1.20/locate/locate.c findutils-4.1.20/locate/locate.c
--- /home/ametzler/TIN/FINDUTILS/findutils-4.1.20/locate/locate.c	Sat Jul 12 16:46:39 2003
+++ findutils-4.1.20/locate/locate.c	Sat Jul 12 16:48:52 2003
@@ -486,7 +486,7 @@
 Usage: %s [-d path | --database=path] [-e | --existing]\n\
       [-i | --ignore-case] [--version] [--help] pattern...\n"),
 	   program_name);
-  fputs (_("\nReport bugs to <bug-findutils@gnu.org>."), stream);
+  fputs (_("\nReport bugs to <bug-findutils@gnu.org>.\n"), stream);
   exit (status);
 }
 
diff -urNad /home/ametzler/TIN/FINDUTILS/findutils-4.1.20/xargs/xargs.c findutils-4.1.20/xargs/xargs.c
--- /home/ametzler/TIN/FINDUTILS/findutils-4.1.20/xargs/xargs.c	Mon May 26 15:39:14 2003
+++ findutils-4.1.20/xargs/xargs.c	Sat Jul 12 16:48:06 2003
@@ -933,6 +933,6 @@
        [--max-args=max-args] [--no-run-if-empty] [--version] [--help]\n\
        [command [initial-arguments]]\n"),
 	   program_name);
-  fputs (_("\nReport bugs to <bug-findutils@gnu.org>."), stream);
+  fputs (_("\nReport bugs to <bug-findutils@gnu.org>.\n"), stream);
   exit (status);
 }
